docs/reference/method/MongoDBGridFSBucket-drop.txt

```diff
@@ -26,4 +26,38 @@ Errors/Exceptions
 
 .. include:: /includes/extracts/error-driver-runtimeexception.rst
 
-.. todo: add examples
+Examples
+--------
+
+.. code-block:: php
+
+   <?php
+
+   $database = (new MongoDB\Client)->test;
+
+   $bucket = (new MongoDB\Client)->test->selectGridFSBucket();
```

Comment:
Since `$database` is assigned above (for listing collections later), let's use `$database->selectGridFSBucket()` on this line.